sql/instance_list.sql

```diff
@@ -109,7 +109,7 @@ INSERT INTO `instance_list` VALUES (59,'nashmeira\'s_plea',72,45,-444,-4,420,127
 -- INSERT INTO `instance_list` VALUES (61,'waking_the_colossus',72,30,0.000,0.000,0.000,0,-1,-1,-1,-1);
 -- INSERT INTO `instance_list` VALUES (62,'zhayolm_remnants',72,100,0.000,0.000,0.000,0,-1,-1,-1,-1);
 -- INSERT INTO `instance_list` VALUES (64,'zhayolm_remnants_ii',72,100,0.000,0.000,0.000,0,-1,-1,-1,-1);
--- INSERT INTO `instance_list` VALUES (65,'arrapago_remnants',72,100,340.000,0.000,-246.000,63,-1,-1,-1,-1);
+INSERT INTO `instance_list` VALUES (65,'arrapago_remnants',72,100,340.000,0.000,-246.000,63,-1,-1,-1,-1);
```

Comment:
i think, in the short term, we should keep this commented out until the last known issues are resolved (especially the double teleport and black screen teleport)

Reply:
Might have been overzealous in listing that as a known issue, more of a possibility. It's only happened to me once when training the entire floor, dualboxing and trying to use a teleporter. Also happened before I changed my despawn code.